webui/src/js/utils/k8s-domain-resource-generator.js

```diff
@@ -270,6 +271,12 @@ define(['models/wkt-project', 'utils/k8s-domain-configmap-generator', 'js-yaml',
         serverPod.resources = resources;
       }
 
+<<<<<<< HEAD
+      if (nodeSelector) {
+        serverPod.nodeSelector = { name: nodeSelector };
+      }
+=======
+>>>>>>> main
       return Object.keys(serverPod).length > 0 ? serverPod : null;
```

Comment:
You do realize that you have committed code with merge conflict markers in it, right?